src/Microsoft.AspNetCore/CertificateLoader.cs

```diff
@@ -178,8 +178,7 @@ public CertificateFileSource(ICertificateFileLoader certificateFileLoader)
             public override X509Certificate2 Load()
             {
                 var certificate = TryLoad(X509KeyStorageFlags.DefaultKeySet, out var error)
-                    ?? TryLoad(X509KeyStorageFlags.UserKeySet, out error)
-                    ?? TryLoad(X509KeyStorageFlags.EphemeralKeySet, out error);
```

Comment:
This isn't supported on in netstandard2.0 and is only available on netcoreapp2.0 cc @javiercn